config/forms/2021_2022.json

```diff
@@ -1111,23 +1123,43 @@
                   "hint_text": "",
                   "type": "text"
                 }
-              }
+              },
+              "depends_on": [{ "startertenancy": "Yes" }]
+            },
+            "fixed_term_tenancy": {
+              "header": "",
+              "description": "",
+              "questions": {
+                "tenancylength": {
+                  "check_answer_label": "Length of fixed-term tenancy",
+                  "header": "What is the length of the fixed-term tenancy to the nearest year?",
+                  "hint_text": "",
+                  "type": "numeric",
+                  "min": 0,
+                  "max": 150,
+                  "step": 1
+                }
+              },
+              "depends_on": [{ "tenancy": "Secure (including flexible)" }, {"tenancy": "Assured Shorthold" }]
             },
-            "letting_provider": {
+            "letting_in_sheltered_accomodation": {
               "header": "",
               "description": "",
               "questions": {
-                "landlord": {
-                  "check_answer_label": "Provider",
-                  "header": "Who is the letting provider?",
+                "letting_in_sheltered_accomodation": {
+                  "check_answer_label": "Is this letting in sheltered accommodation?",
+                  "header": "Is this letting in sheltered accommodation?",
                   "hint_text": "",
                   "type": "radio",
                   "answer_options": {
-                    "0": "This landlord",
-                    "1": "Another registered provider - includes housing association or local authority"
+                    "0": "Yes - sheltered housing",
+                    "1": "Yes - extra care housing",
+                    "2": "No",
+                    "3": "Don't know"
```

Comment:
Add divider

Reply:
I'm not quite sure what you mean here, do you have an example?

Reply:
I think it’s this… adds an ‘or’ between the two options

```
"2": "No",
"divider": true,
"3": "Don’t know"
```

Reply:
<img width="501" alt="Screenshot 2022-01-13 at 15 18 22" src="https://user-images.githubusercontent.com/47317567/149357387-cf17411a-18d3-4d19-9d31-67a063655a22.png">

Is this what you had in mind?

Reply:
Yup! Thanks!

Reply:
That LGTM